src/acvp.h

```diff
@@ -1032,13 +1163,53 @@ ACVP_RESULT acvp_enable_rsa_cap_parm(
                             int value
                             );
 
+/*! @brief acvp_enable_rsa_bignum_parm() allows an application to specify
+       BIGNUM operational parameters to be used for a given RSA alg during a
+       test session with the ACVP server.
+
+    This function behaves the same as acvp_enable_rsa_cap_parm() but instead
+    allows the application to specify a BIGNUM parameter
+
+    @param ctx Address of pointer to a previously allocated ACVP_CTX.
+    @param cipher ACVP_CIPHER enum value identifying the crypto capability.
+    @param mode ACVP_RSA_MODE enum value specifying mode. An example would be
+        ACVP_RSA_MODE_KEYGEN
+    @param param ACVP_RSA_PARM enum value identifying the algorithm parameter
+       that is being specified.  An example would be public exponent
+    @param value BIGNUM value corresponding to the parameter being set
+
+    @return ACVP_RESULT
+ */
 ACVP_RESULT acvp_enable_rsa_bignum_parm (ACVP_CTX *ctx,
                              ACVP_CIPHER cipher,
                              ACVP_RSA_MODE mode,
                              ACVP_RSA_PARM param,
                              BIGNUM *value
                            );
 
+/*! @brief acvp_enable_rsa_primes_parm() allows an application to specify
```

Comment:
Do the function headers with leading '*' turn out the same as those without.
Probably want to be consistent if it impacts the end document.

Reply:
Removed the extras :)